### PR TITLE
Release 1.2.0

### PR DESCRIPTION
**Pull request by daviwil**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,30 @@
...
+- [PowerShell/vscode-powershell#771](https://github.com/PowerShell/vscode-powershell/issues/771) -
+  Improved dynamic comment help snippet performance in scripts with many functions
+
+- [PowerShell/vscode-powershell#774](https://github.com/PowerShell/vscode-powershell/issues/774) -
+  Pressing Enter now causes custom prompt functions to be fully evaluated
+
+- [PowerShell/vscode-powershell#770](https://github.com/PowerShell/vscode-powershell/issues/770) -
+  Fixed issue where custom prompt function might be written twice when
+  starting the integrated console
+
```
Comment (rkeithhill) [inline CHANGELOG.md]: Do we want to mention that the Session Manager now lists installed versions of PowerShell Core on Windows and that the extension defines a reusable problem matcher for Pester named `$pester` (requires the Pester parameter `-PesterOption @{IncludeVSCodeMarker=$true}`)?

Comment (daviwil) [inline CHANGELOG.md]: That's mentioned in the vscode-powershell changelog, not applicable for PSES :)

Comment (rkeithhill) [inline CHANGELOG.md]: Might also want to mention there is a known issue with Pester problem matcher and tasks version 2.0.0.

Comment (rkeithhill) [inline CHANGELOG.md]: Duh. Not enough caffeine yet.  :-)

Comment (daviwil) [inline CHANGELOG.md]: I'll add that too, thanks!

**Pull request merged**